x-pack/test/security_solution_cypress/cypress/tasks/alerts.ts

```diff
@@ -387,26 +389,24 @@ export const showTopNAlertProperty = (propertySelector: string, rowIndex: number
 };
 
 export const waitForAlerts = () => {
-  /*
-   * below line commented because alertpagefiltersenabled feature flag
-   * is disabled by default
-   * target: enable by default in v8.8
-   *
-   * waitforpagefilters();
-   *
-   * */
   waitForPageFilters();
   cy.get(REFRESH_BUTTON).should('not.have.attr', 'aria-label', 'Needs updating');
   cy.get(DATAGRID_CHANGES_IN_PROGRESS).should('not.be.true');
   cy.get(EVENT_CONTAINER_TABLE_LOADING).should('not.exist');
   cy.get(LOADING_INDICATOR).should('not.exist');
+  cy.waitForNetworkIdle(2000);
```

Comment:
are we sure we want this static 2 seconds here? What if 2 seconds isn't enough in some scenarios, what if it's too much?

Reply:
The way `cypress-network-idle` package waits until it detects that all network requests have run, and then gives the prescribed amount of time after that to determine that no more requests are made. 

It was a package created externally to cypress based on the thread here for reference: https://github.com/cypress-io/cypress/issues/1773

Reply:
I understand, but my comment was on the 2000ms value, why that value? Are we sure it's enough? Could it be too much? Where does the 2000 come from?

Also, does this mean that if tomorrow we make subsequent calls in some of our pages, calls that would for example load things in the background... things that would not necessary have to do with the content of the page that a test would care about, we would just wait longer before running these tests?
I guess if it works now maybe that is a problem for tomorrow us, but I can't stop thinking it would be better to wait for specific api calls that we care about no? We shouldn't care that all the network requests have run...

Reply:
> Also, does this mean that if tomorrow we make subsequent calls in some of our pages, calls that would for example load things in the background... things that would not necessary have to do with the content of the page that a test would care about, we would just wait longer before running these tests?

Yep, great point here, this is a heavy handed approach. I'll update this to take a parameter to define the api endpoint you would like to wait on, (i.e. the endpoint for the table, or the visualizations, etc...) And only call it if that endpoint is provided?